src/main/docker/docker-compose.yml

```diff
@@ -77,83 +77,274 @@ services:
+      - rabbitmq.instances.jsonlist=${REMREM_PUBLISH_RABBITMQ_INSTANCES_LIST}
+      - generate.server.uri=http://generate:8080
+      - generate.server.path=
+      - activedirectory.publish.enabled=false
+      - activedirectory.ldapUrl=
+      - activedirectory.managerPassword=
+      - activedirectory.managerDn=
+      - activedirectory.rootDn=
+      - activedirectory.userSearchFilter=
+      - logging.level.root=OFF
+      - logging.level.org.springframework.web=ERROR
+      - logging.level.com.ericsson.eiffel.remrem.producer=INFO
+      - WAIT_MB_HOSTS=rabbitmq:15672
+    networks:
+      eiffel_2.0_1:
+        aliases:
+          - publish
+
+
+  jenkins:
+    restart: always
+    image: bitnami/jenkins:2.138.3
+    expose:
+      - "8080"
+    ports:
+      - "8082:8080"
+      - "50001:50000"
+    depends_on:
+      - rabbitmq
+      - mongodb
+    environment:
+      - JAVA_OPTS="-Djenkins.install.runSetupWizard=false"
+      - JENKINS_USERNAME=admin
+      - JENKINS_PASSWORD=admin
+    networks:
+      eiffel_2.0_1:
+        aliases:
+          - jenkins
+
+
+  ############### Start EI backend artifact rules #############################
+  ei-backend-artifact:
+    restart: always
+    image: eiffelericsson/eiffel-intelligence-backend:1.0.1
+    expose:
+      - "8080"
+    ports:
+      - "8070:8080"
+    depends_on:
+      - mongodb
+      - rabbitmq
+    networks:
+      eiffel_2.0_1:
+        aliases:
+          - ei-backend-artifact
+    environment:       # Overrides settings in application config file
+      - SpringApplicationName=ei-backend-artifact
+      - server.port=8080
+      - rules.path=/ArtifactRules.json
+      - rabbitmq.host=rabbitmq
+      - rabbitmq.port=5672
+      - rabbitmq.domainId=ei-domain
+      - rabbitmq.componentName=eiffel-intelligence-artifact
+      - rabbitmq.waitlist.queue.suffix=waitlist-artifact
+      - rabbitmq.exchange.name=ei-exchange
+      - rabbitmq.user=myuser
+      - rabbitmq.password=myuser
+      - spring.data.mongodb.host=mongodb
+      - spring.data.mongodb.port=27017
+      - spring.data.mongodb.database=eiffel-intelligence-artifact
+      - missedNotificationDataBaseName=MissedNotification-artifact
+      - search.query.prefix=object
+      - aggregated.object.name=aggregatedObject
+      - spring.mail.properties.mail.smtp.auth=false
+      - spring.mail.properties.mail.smtp.starttls.enable=false
+      - er.url=http://${HOST}:8084/search/
+      - logging.level.root=OFF
+      - logging.level.org.springframework.web=ERROR
+      - logging.level.com.ericsson.ei=ERROR
+      - WAIT_MB_HOSTS=rabbitmq:15672
+      - WAIT_DB_HOSTS=mongodb:27017
+
+  ############### Stop EI backend artifact rules ##############################
+
+
+  ############### Start EI backend sourcechange rules ##########################
+  ei-backend-sourcechange:
     restart: always
-    image: eiffelericsson/eiffel-intelligence-backend:1.0.0-1
+    image: eiffelericsson/eiffel-intelligence-backend:1.0.1
```

Comment:
If its mandatory already now to execute env.bash before running docker-compose, then you could put all image versions in env.bash,, then we have all version collected in one place.

Reply:
or you can put the whole image tag inclusive version in env.bash.

Reply:
I've moed out the ports to the services to env.bash, and I'll move out the image versions also